addons/slack/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "engines": {
+        "node": "10"
+    },
+    "dependencies": {
+        "@fusebit/add-on-sdk": "^1.3.1",
```

Comment:
1.0.6 in some packages, 1.3.1 here?

Reply:
All the Slack stuff should be using 1.3.1, I haven't gone to update the Atlassian and Google connectors